### PR TITLE
Updated DBXrefs for PECO issue trackers

### PR DESCRIPTION
**Pull request by cooperl09**

### CODE REVIEW

File: metadata/db-xrefs.yaml [hunk 1]
```diff
@@ -2042,6 +2052,16 @@
       url_syntax: http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=[example_id]
       example_id: PDB:1A4U
       example_url: http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1A4U
+- database: PECO_GIT
+  name: GitHub
```
Comment (kltm) [inline metadata/db-xrefs.yaml]: @cooperl09 Apologies, but could you supply a name besides "GitHub" here?

**Comment by cooperl09** [comment]
Sure, I will make all these more explicit.

**Comment by cooperl09** [comment]
Sure, I will make all these more explicit.

**Comment by cooperl09** [comment]
Sure, I will make all these more explicit.

**Pull request closed without merging**